internal/turso/databases.go

```diff
@@ -25,20 +25,30 @@ type Database struct {
 	IsSchema      bool `json:"is_schema"`
 }
 
-type DatabasesClient client
-
-func (d *DatabasesClient) List(options map[string]string) ([]Database, error) {
-	path := d.URL("")
+type DatabaseListOptions struct {
+	Group  string `json:"group,omitempty"`
+	Schema string `json:"schema,omitempty"`
+}
 
+func (o DatabaseListOptions) Encode() string {
 	query := url.Values{}
-	for key, value := range options {
-		if value != "" {
-			query.Add(key, value)
-		}
+	if o.Group != "" {
+		query.Set("group", o.Group)
+	}
+	if o.Schema != "" {
+		query.Set("schema", o.Schema)
 	}
+	return query.Encode()
+}
+
+type DatabasesClient client
+
+func (d *DatabasesClient) List(options DatabaseListOptions) ([]Database, error) {
+	path := d.URL("")
 
-	if len(query) > 0 {
-		path += "?" + query.Encode()
+	encodedOptions := options.Encode()
+	if encodedOptions != "" {
+		path += "?" + encodedOptions
```

Comment:
⛏️: totally style. If you're into one-liners and reusing names 😎 
```suggestion
	if options := options.Encode(); options != "" {
		path += "?" + options
```